test/unit/xml_state_test.py

```diff
@@ -181,6 +181,15 @@ def test_add_repository(self):
         assert self.state.xml_data.get_repository()[3].get_priority() == 1
         assert self.state.xml_data.get_repository()[3].get_imageinclude() is True
 
+    def test_add_repository_with_empty_values(self):
+        self.state.add_repository('repo', 'type', '', '', True)
+        assert self.state.xml_data.get_repository()[3].get_source().get_path() \
+            == 'repo'
+        assert self.state.xml_data.get_repository()[3].get_type() == 'type'
+        assert self.state.xml_data.get_repository()[3].get_alias() == ''
+        assert self.state.xml_data.get_repository()[3].get_priority() == None 
```

Comment:
sure, it should be fixed now, I forgot to verify style with `tox -e check` before making the PR.